tests/scripts/set-release-ver.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+TO_TAG="$1"
+
+FROM_TAG=$(grep "docker.io/rook/ceph" deploy/examples/images.txt | awk -F : '{ print $2 }')
+
+scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+SED="${scriptdir}/../../build/sed-in-place"
+
+# This script is supposed to facilitate the preparation of a rook release by
+# changing example and documentation files to use the new release version (TO_TAG)instead the old one (FROM_TAG).
+#
+# It will usually be run on a release branch
+
+FILES_DOCS=(
+ Documentation/Getting-Started/quickstart.md
+ Documentation/Storage-Configuration/Monitoring/ceph-monitoring.md
+ Documentation/Upgrade/rook-upgrade.md
+)
+FILES_CHARTS=(
+ deploy/charts/rook-ceph/values.yaml
+)
+FILES_EXAMPLES=(
+ deploy/examples/*.yaml
+ deploy/examples/images.txt
+)
+
+$SED -e "s/rook\/ceph:$FROM_TAG/rook\/ceph:$TO_TAG/g" -e "s/rook-version=$FROM_TAG/rook-version=$TO_TAG/g" Documentation/Upgrade/rook-upgrade.md
```

Comment:
It's easier to read separate sed expressions when they're broken up to one expression per line

```suggestion
$SED -e "s/rook\/ceph:$FROM_TAG/rook\/ceph:$TO_TAG/g" \
     -e "s/rook-version=$FROM_TAG/rook-version=$TO_TAG/g" \
  Documentation/Upgrade/rook-upgrade.md
```